app/Question.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Question extends Model
+{
+	/**
+	*	Attributes to be mass filled.
+	*/
+    protected $fillable = [
+        'user_id', 'organization_id', 'event_id', 'question', 
+        'question_body', 'answer', 'answered_at'
+    ];
+    /**
+    *	Attribute(s) not to be mass filled.
+    */
+    protected $guarded = array('id');
+
+    /**
+    *	A question belongs to a single event.
+    */
+    public function event()
+    {
+    	return $this->belongsTo('Event');
+    }
+
+    /**
+    *	Fetches the answered questions that would be presented in the events page.
+    */
+    public function showQuestions()
+    {
+    	// may be changed to take(10) depending on the layout of the 'event' view.
+    	return DB::table('questions')
+    					->whereNotNull('answer')
+    					->get();
+    }
+
```

Comment:
No need for the show methods because the controller will perform the query on the Eloquent model. However, you may set a scope function to fetch answered questions only. For example
```public function scopeAnswered($query)
    {
        $query->whereNotNull('answer');
    }```